packages/core-mobile/app/new/features/portfolio/hooks/useAvailableNativeTokenBalanceForNetworkAndAccount.ts

```diff
@@ -39,7 +39,11 @@ export function useAvailableNativeTokenBalanceForNetworkAndAccount(
 
     // Handle XP-style tokens (PVM/AVM)
     if (
+      // TODO: fix type mismatch after fully migrating to the new backend balance types
+      // @ts-ignore
```

Comment:
these ts-ignore is just a temp solution for now (otherwise we will have to adjust 100+ files). eventually, we will need to migrate the code to use the new data type from the balance service, which means we will not need NormalizedBalancesForAccount or AdjustedNormalizedBalancesForAccount type anymore.